### PR TITLE
Add openvpn

### PR DESCRIPTION
**Pull request by johnfosborneiii**
#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

Passes all unit testing with provided samples

### CODE REVIEW

File: openvpn/vendor/down.sh
Comment (rawlingsj) [inline openvpn/vendor/down.sh:1]: Could we add a link to the source of these vendored files?  It helps to help maintain them if they change upstream.

**Review by rawlingsj — DISMISSED** [review]

File: openvpn/vendor/openvpn.initd
[the remark this answers is not in the document]
Comment (johnfosborneiii) [reply openvpn/vendor/openvpn.initd:1]: It's more of a placeholder, I can remove them for now

File: openvpn/vendor/down.sh
Comment (johnfosborneiii) [reply openvpn/vendor/down.sh:1]: https://git.alpinelinux.org/aports/tree/main/openvpn/

File: openvpn/vendor/openvpn.initd
[the remark this answers is not in the document]
Comment (johnfosborneiii) [reply openvpn/vendor/openvpn.initd:1]: Should I just remove them from the main package and keep them in the subpackage?

**Review by kaniini — APPROVED** [review]

**Pull request merged**